apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/interceptor/enhance/InstMethodsInterWithOverrideArgs.java

```diff
@@ -70,12 +75,16 @@ public Object intercept(@This Object obj, @AllArguments Object[] allArguments, @
         @Morph OverrideCallable zuper) throws Throwable {
         EnhancedInstance targetObject = (EnhancedInstance) obj;
 
+        long interceptorTimeCost = 0L;
+        long beforeStartTime = System.nanoTime();
```

Comment:
```suggestion
        long startTimeOfMethodBeforeInter = System.nanoTime();
```

Let's use this naming style. `beforeStartTime` like `before start time`, which is not our meaning.